README.md

```diff
@@ -1,11 +1,131 @@
 # irctest
 
-This is the integration test suite for Oragono, ultimately derived from [ProgVal/irctest](https://github.com/ProgVal/irctest), which is a general-purpose IRC protocol compatibility testing suite.
+This project aims at testing interoperability of software using the
+IRC protocol, by running them against test suites and making different
+software communicate with each other.
 
-Some of these tests may be applicable to other projects (we attempt to mark the tests that are only applicable to Oragono).
+It is very young and does not contain a lot of test cases yet.
 
-This suite needs more test cases. Contributions are welcome and are a great way to help the Oragono project!
+## The big picture
 
-## Running Tests
+This project contains:
+
+* IRC protocol test cases
+* small wrappers around existing software to run tests on them
+
+Wrappers run software in temporary directories, so running `irctest` should
+have no side effect, with [the exception of Sopel](https://github.com/sopel-irc/sopel/issues/946).
+
+## Prerequisites
+
+Install irctest and dependencies:
+
+```
+git clone https://github.com/ProgVal/irctest.git
+cd irctest
+pip3 install --user -r requirements.txt pyxmpp2-scram
```

Comment:
requirements.txt is deleted, including in this branch.

How are we thinking about deps going forwards? I'd prefer that this be entirely self-contained, with the exception of pytest, which can typically be obtained from a system package manager. (Would you be open to vendoring pyxmpp2-scram?)

Reply:
Oh, ok.

Yes, we could vendor pyxmpp2-scram, it's not that big. We'll see if we ever need larger dependencies. For example if we want to add a test for ecdsa, that would b way too big for vendoring, but you're not there yet. And we can make them optional, anyway.